platform/android/java/editor/build.gradle

```diff
@@ -154,4 +144,37 @@ android {
             doNotStrip '**/*.so'
         }
     }
+
+    flavorDimensions = ["vendor"]
+    productFlavors {
+        google {
+            dimension "vendor"
+            missingDimensionStrategy 'products', 'editor'
+        }
+        meta {
+            dimension "vendor"
+            missingDimensionStrategy 'products', 'editor'
+            ndk {
+                //noinspection ChromeOsAbiSupport
```

Comment:
Should this be removed, or is this something we plan to enable later?

Reply:
That's a [lint warning suppressor](https://stackoverflow.com/a/42060262) used to suppress warnings about missing Chrome OS support.